commands/proposal_submission.go

```diff
@@ -395,6 +407,17 @@ func checkUpdateMarketChanges(change *protoTypes.ProposalTerms_UpdateMarket) Err
 	}
 
 	changes := change.UpdateMarket.Changes
+	lppr, err := num.DecimalFromString(changes.LpPriceRange)
+	if err != nil {
+		errs.AddForProperty("proposal_submission.terms.change.update_market.changes.lp_price_range", ErrIsNotValidNumber)
+	} else {
+		if lppr.IsNegative() || lppr.IsZero() {
```

Comment:
same here

Reply:
Not a fan of `else { if {`, perhaps using `else if`?

Reply:
Sure, why not, changed now.